app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -119,29 +168,37 @@ You can work in an English school and, alongside your job, apply for assessment
 
 ### Apply for QTS in England (from spring 2023)
 
-From spring 2023, qualified teachers from Ukraine will be able to apply to the Teaching Regulation Agency for QTS (if they meet the criteria) without paying a fee to a training provider. [Find out more about the way QTS will be awarded to teachers from overseas](https://www.gov.uk/government/publications/awarding-qualified-teacher-status-to-overseas-teachers).
+From spring 2023, qualified teachers from Ukraine will be able to apply to the Teaching Regulation Agency for QTS (if they meet the criteria) without paying a fee to a training provider. [Learn more about the way QTS will be awarded to teachers from overseas](https://www.gov.uk/government/publications/awarding-qualified-teacher-status-to-overseas-teachers).
 
 ### Salaries and benefits
 
 Your salary as a teacher depends on whether you have QTS, and your level of experience. The salary for a teacher without QTS will generally start at £18,419 outside London. [Find out more about salaries and benefits](/salaries-and-benefits) for teachers in England.
 
 ## Train to teach in England
 
-If you are resident in the UK under:
+From 1 August 2022, you will have the right to apply to train to teach in England if you are resident in the UK under:
 
 * [the Ukraine Family Scheme visa](https://www.gov.uk/guidance/apply-for-a-ukraine-family-scheme-visa)
 * [the Ukraine Sponsorship Scheme](https://www.gov.uk/guidance/apply-for-a-visa-under-the-ukraine-sponsorship-scheme)
 * [the Ukraine Extension Scheme](https://www.gov.uk/guidance/apply-to-stay-in-the-uk-under-the-ukraine-extension-scheme)
 
-From 1 August 2022, you will:
+### Deadlines for applications
+
+The closing date for applications to train to teach in 2022/23 is 6 September 2022. Applications for 2023/24 open on 11 October 2022.
+
+If you miss the deadline to apply this year, you could consider finding work as a [teaching assistant](#apply-for-a-teaching-job) to gain valuable experience for your teacher training course. 
```

Comment:
@laurahermionetennant92 shouldn't really have the same link text that links to different places (bad for SEO and accessibility) - either remove the link on teaching assistant or link to the other teaching assistant page you linked to earlier on?

Reply:
done